extensions/integration-tests/src/index.ts

```diff
@@ -19,6 +22,36 @@ const options: any = {
 
 if (suite === SuiteType.Stress) {
 	options.timeout = 7200000;	// 2 hours
+	if (process.env.StressRuntime) {
```

Comment:
the indention seems to be off

Reply:
it might have been, but currently it is all good and done by 'Format Document' so it is good.